packages/office-ui-fabric-react/src/components/Layer/examples/Layer.Basic.Example.tsx

```diff
@@ -1,29 +1,27 @@
-import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
-import 'office-ui-fabric-react/lib/common/_exampleStyles.scss';
-import { Layer } from 'office-ui-fabric-react/lib/Layer';
-import { AnimationClassNames } from 'office-ui-fabric-react/lib/Styling';
-import { BaseComponent, css } from 'office-ui-fabric-react/lib/Utilities';
 import * as PropTypes from 'prop-types';
 import * as React from 'react';
 import * as styles from './Layer.Example.scss';
+import { AnimationClassNames } from 'office-ui-fabric-react/lib/Styling';
+import { BaseComponent, css } from 'office-ui-fabric-react/lib/Utilities';
+import { Layer } from 'office-ui-fabric-react/lib/Layer';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import 'office-ui-fabric-react/lib/common/_exampleStyles.scss';
 
-export interface ILayerContentExampleState {
-  time: string;
+interface ILayerBasicExampleContext {
+  message: string | undefined;
```

Comment:
nit: Our convention is generally `message?: string`

Reply:
Good catch; thanks.